data/human/human missions.txt

```diff
@@ -2502,6 +2502,123 @@ event "terraforming follow-up"
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					decline
+			`	You're able to contact Amy. She has been continuing to monitor the status of Tundra's climate, alongside the researchers from the Deep who she worked with to terraform this world. You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them and agrees to meet you in the spaceport bar.`
+			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`
+			`	She has a slew of questions ranging from how the Wanderers' determine a planet's identity to "How often are asteroids used in their work?" Unfortunately, you are not able to offer much information about the specific details of how the Wanderers go about their terraforming work. Disappointed, but not defeated, Amy asks, "Could you take me to visit these aliens?`
+			`	Now that the Kor Mereti drones are peaceful, and the Kor Sestor drones have been eliminated, there is no longer an immediate threat to the Wanderers, and they will be focused much more on their work to repair the former Korath worlds. They would probably be happy to discuss some of that work with someone as enthused by the prospect as Amy.`
+			choice
+				`	"Alright. I'll prepare a bunk for you."`
+				`	"Sorry, that's too far out of my way."`
+					decline
+			`	She is delighted, and explains that she will meet you at your ship later today, before leaving to attend to her work in preparation for her absence.`
+				accept
+	on complete
+		conversation
+			`As you approach <origin>, Amy is astounded by the magnitude of volcanic ash in the air, far outweighing what was released by the volcano you triggered on Tundra. "This place makes Tundra seem temperate," Amy remarks after observing the glaciers that occupy most of the planet's polar regions.`
+			`	Her fascination with the state of this world is temporarily forgotten when she exits your ship and sees the Wanderers for the first time. "I don't know what I was expecting them to be like, but it wasn't birds." However, she quickly returns to the reason for coming here. "Do you know how we can find someone who can answer my questions?"`
+			`	You approach one of the nearby Wanderers and ask if they could direct you to Tema'a Iriket, the terraforming director. They don't appear to understand the Hai language, but they seem to recognize the name and gesture for you to wait in the spaceport, before heading off in a different direction, presumably to fetch Iriket for you.`
```

Comment:
```suggestion
			`	Approaching one of the nearby Wanderers, you ask if they could direct you to Tema'a Iriket, the terraforming director, though they don't appear to understand the Hai language from your translator. Still, they seem to recognize the name and gesture for you to wait in the spaceport, flying off into the distance presumably to fetch Iriket for you.`
```

Reply:
Why do you suggest rearranging the sentences in this way?
I don't think the part about the Wanderer not understanding the Hai translator fits better with the preceding clause.
First, I think it more closely relates to the following clause, since that is about how we overcome that issue, following on directly from it.
Second, grouping it with the preceding clause in one sentence gives me a sense of concurrency, we ask them though they don't understand what we say, but we aren't going to know that they don't understand until after we've asked, or at least, until after outr character is done speaking and the translator begins. It gives the sense that we ask despite them not understanding, but we haven't gone into it knowing that they won't understand. I think it works better in the next sentence where it becomes more like "they recognise the name despite not understanding the Hai language."

Additionally, I think the "before" is important in the last part because, as you have it, it seems like they are flying off as they gesture towards the spaceport, which I don't think works very well. I'm envisioning them gesture and then fly off; two separate actions.

I have applied some of the other adjustments here, though.